specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json

```diff
@@ -1406,7 +1406,21 @@
     },
     "EnvironmentUpdateParameters": {
       "type": "object",
+      "discriminator": "kind",
       "properties": {
+        "kind": {
+          "type": "string",
+          "readOnly": true,
```

Comment:
`readonly` property cannot exist in the request of examples.

Reply:
> `readonly` property cannot exist in the request of examples.

Then how can I discriminate polymorph?

Reply:
Please remove it from request in the example files.